SimG4Core/Application/src/RunManagerMTWorker.cc

```diff
@@ -108,16 +108,16 @@ struct RunManagerMTWorker::TLSData {
   std::unique_ptr<CustomUIsession> UIsession;
   std::unique_ptr<RunAction> userRunAction;
   std::unique_ptr<SimRunInterface> runInterface;
-  SimActivityRegistry registry;
+  std::unique_ptr<SimActivityRegistry> registry;
```

Comment:
And actually why is `registry` changed to a `unique_ptr`? I don't see any reason for that as it is always constructed, and the size of `TLSData` does not count towards the maximum TLS size.

Reply:
@makortel , this is for uniformity, most of other ptr in this data struct also definitely constructed.